packages/metadata-crawler/src/crawler.ts

```diff
@@ -41,60 +49,60 @@ async function addVerificationClaimToDatabase(address: string, domain: string, v
 
     await client
       .query(query, values)
-      .catch((error) => console.error(`Database error ${error}\n query: ${query}`))
-      .then(() => console.log(`Verification flag added to domain ${domain} and address ${address}`))
+      .catch((error) => logger.error(`Database error ${error}\n query: ${query}`))
+      .then(() => logger.info(`Verification flag added to domain ${domain} and address ${address}`))
   } catch (err) {
-    console.error('Error updating the database', err)
+    logger.error('Error updating the database', err)
   }
 }
 
 async function handleItem(item: { url: string; address: string }) {
   try {
     let metadata = await IdentityMetadataWrapper.fetchFromURL(item.url)
     let claims = metadata.filterClaims(ClaimTypes.DOMAIN)
+    const accounts = metadata.filterClaims(ClaimTypes.ACCOUNT)
 
     const numClaims = claims.length
     for (let i = 0; i < numClaims; i++) {
       const claim = claims[i]
-      const alreadyVerified = await isClaimAlreadyVerified(item.address, claim.domain)
-      if (!alreadyVerified) {
-        console.log(`Verifying ${claim.domain} for address ${item.address}`)
-        const verified = await verifyDomainRecord(
-          JSON.parse(metadata.toString()).meta.signature,
-          item.address,
-          claim.domain,
-          metadata
-        ).catch((error) => console.error(`Error in verifyDomainRecord ${error}`))
-        console.debug(`Claim = ${claim.domain}`)
-        if (verified === undefined)
-          await addVerificationClaimToDatabase(item.address, claim.domain, true)
-        else console.error(verified)
-      }
+      const addressWith0x = '0x' + item.address
+      logger.debug('Claim: ' + serializeClaim(claim))
+      logger.debug('Accounts: ' + JSON.stringify(accounts))
+      // const alreadyVerified = await isClaimAlreadyVerified(item.address, claim.domain)
+      // logger.debug(`Is already verified? ${alreadyVerified}`)
+      // if (!alreadyVerified) {
+      logger.debug(`Verifying ${claim.domain} for address ${addressWith0x}`)
+
+      const verified = await verifyDomainRecord(addressWith0x, claim).catch((error) =>
+        logger.error(`Error in verifyDomainClaim ${error}`)
+      )
+      if (verified === undefined)
+        await createVerificationClaims(item.address, claim.domain, true, accounts)
```

Comment:
I thought verified indicates success, so shouldn't we only do this when this is true?

Reply:
Used same approach used in other places in contractKit. Keep in that way for consistency. 
I renamed in any case the variable for clarity